src/conn/api_calc_modify.c

```diff
@@ -124,7 +129,6 @@ wiredtiger_calc_modify(WT_SESSION *wt_session,
 	cms.e2 = cms.s2 + newv->size;
 	cms.maxdiff = maxdiff;
 	cms.maxentries = *nentriesp;
-	*nentriesp = 0;
```

Comment:
Moving this introduces a bug.  `*nentriesp` is an in/out parameter.  The input value needs to be copied before it is overwritten.  We don't make any promises about the return value when the call fails, so I don't see any reason to move it.

Reply:
Ouch, sorry -- reverted.